.github/workflows/test_dockerfiles.yml

```diff
@@ -0,0 +1,38 @@
+name: Test Dockerfiles
+
+on: [push, pull_request]
+
+jobs:
+  lint_dockerfiles:
+    name: Lint Dockerfiles
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Hadolint
+        run: |
+          cd "${HOME}";
+          mkdir bin;
+          wget https://github.com/hadolint/hadolint/releases/download/v2.8.0/hadolint-Linux-x86_64 ;
```

Comment:
I was doing myself this same thing, I was wondering if we need to cache, because is not a small file and people may get upset if we are pulling on every push/pr.

Reply:
I've added caching the executable, but I'm not sure it's so useful since it's only a ~5MB executable downloaded from GitHub. Maybe only running that workflow when dockerfiles are modified would be better

Reply:
Well just not to annoy people. I think it should be free from github to github. But who knows.